engine/consensus/dkg/reactor_engine.go

```diff
@@ -249,10 +249,10 @@ func (e *ReactorEngine) startDKGForEpoch(currentEpochCounter uint64, first *flow
 //
```

Comment:
would suggest to update the beginning of the godoc 
https://github.com/onflow/flow-go/blob/550fd3f739237b9fb241f13a69b19de5b7ce56b5/engine/consensus/dkg/reactor_engine.go#L247-L248
to 
```golang
// handleEpochCommittedPhaseStarted is invoked upon the transition to the EpochCommitted
// phase, when the canonical beacon key vector is incorporated into the protocol state.
// Alternatively we invoke this function preemptively on startup if we are in the
// EpochCommitted Phase, in case the `EpochCommittedPhaseStarted` event was missed
// due to a crash.
```